groups/stack/module-cloudwatch/variables.tf

```diff
@@ -0,0 +1,26 @@
+//----------------------------------------------------------------------
+// Standard Variables
+//----------------------------------------------------------------------
+variable "environment" {
+  description = "The name of the environment this cluster is part of e.g. live, staging, dev. etc."
+  type        = string
+}
+
+# ------------------------------------------------------------------------------
+# AWS Variables
+# ------------------------------------------------------------------------------
+variable "aws_region" {
+  type        = string
+  description = "The AWS region in which resources will be administered"
+}
+
+#
```

Comment:
I would stick with a consistent comment prefix in this file - `#` or `//`. At the mo in this file there's both. Not sure why TF has 2 options TBH.

(actually it's the same in mine too - which I will now change also!)
Interesting what you spot in a PR but overlook in your own TF 🙄

Reply:
Dangers of "copy and paste". I have used # since that seems the main way we comment lines in our Terraform projects